frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,38 @@
+import { Avatar } from '~/libs/components/avatar/avatar.js';
+import { Button } from '~/libs/components/components.js';
+import { useAppSelector } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+const mockedUser = {
+  id: 1,
+  email: 'test@gmail.com',
+  firstName: 'John',
+  lastName: 'Doe',
+};
+
+const Profile: React.FC = () => {
+  const { user } = useAppSelector(({ auth }) => ({
+    user: auth.user,
+  }));
+
+  const currentUser = user ?? mockedUser;
+  const userName = `${currentUser.firstName} ${currentUser.lastName}`;
+
+  return (
+    <div className={styles.profile}>
+      <div className={styles.userInfoBlock}>
+        <div className={styles.avatar}>
+          <Avatar username={userName} avatarUrl={null} />
```

Comment:
In case if you want to add custom styles for `Avatar`
```suggestion
          <Avatar className={styles.avatar} username={userName} avatarUrl={null} />
```